armory/datasets/package.py

```diff
@@ -0,0 +1,124 @@
+from pathlib import Path
+import os
+import random
+import shutil
+import string
+import subprocess
+
+from armory.logs import log
+from armory.datasets import build, common
+
+
+def package(
+    name,
+    version: str = None,
+    data_dir: str = None,
+    cache_subdir=common.CACHE_SUBDIR,
```

Comment:
`cache_subdir` kwarg isn't being used

Reply:
i think the information it contains is captured in the `common.get_cache_dataset_path()` call

Reply:
Good catch, I refactored that into common